### PR TITLE
M0078DDG-502 Update Tutorial Understand How To build a Firmware

### PR DESCRIPTION
**Pull request by thomasponsmicroej**
- Remove "MicroEJ" when possible
- Rewording of some sentences
- Typos

### CODE REVIEW

File: Tutorials/tutorialUnderstandMicroEJFirmwareBuild.rst [hunk 1]
```diff
@@ -1,130 +1,124 @@
...
-* the MicroEJ Architecture and MicroEJ Packs
-* the Abstraction Layers implementations
-* the MicroEJ Simulator and its associated MicroEJ Mocks
+* the Architecture and :ref:`pack_overview`,
+* the Abstraction Layers implementations,
+* the Simulator and its associated Mocks,
+* a C Board Support Package (BSP) with C drivers and an optional RTOS.
 
-The platform .zip files contain:
+The Platform sources consists in the following projects:
```
Comment (tdelhomenie-microej) [inline Tutorials/tutorialUnderstandMicroEJFirmwareBuild.rst:61]: I would keep "contains".

File: Tutorials/tutorialUnderstandMicroEJFirmwareBuild.rst [hunk 2]
```diff
@@ -1,130 +1,124 @@
...
+~~~~~~~~~~~
 
-A MicroEJ Application is a Java project that can be configured (in the
+An Application is a Java project that can be configured (in the
 :guilabel:`Run configurations ...` properties):
 
 1. to either run on:
 
-  * a simulator (computer desktop),
+  * a the Simulator (computer desktop),
```
Comment (tdelhomenie-microej) [inline Tutorials/tutorialUnderstandMicroEJFirmwareBuild.rst:83]:
```suggestion
  * the Simulator (computer desktop),
```

File: Tutorials/tutorialUnderstandMicroEJFirmwareBuild.rst [hunk 3]
```diff
@@ -139,119 +133,110 @@ See :ref:`module_repository` for more information.
...
 
-The next schema presents the steps to build a MicroEJ Mono-Sandbox
-Firmware (previously known as MicroEJ Single-app Firmware).  The steps
+The next schema presents the build flow of a Mono-Sandbox
+Firmware (previously known as a MicroEJ Single-app Firmware). The steps
 are:
 
 
-1. Build the MicroEJ Application into a ``microejapp.o`` using MicroEJ SDK
+1. Build the Application using MicroEJ SDK (generates a ``microejapp.o`` file).
```
Comment (tdelhomenie-microej) [inline Tutorials/tutorialUnderstandMicroEJFirmwareBuild.rst:184]:
```suggestion
1. Build the Application using the SDK (generates a ``microejapp.o`` file).
```

File: Tutorials/tutorialUnderstandMicroEJFirmwareBuild.rst [hunk 4]
```diff
@@ -139,119 +133,110 @@ See :ref:`module_repository` for more information.
...
 
-Build a MicroEJ Firmware
-~~~~~~~~~~~~~~~~~~~~~~~~
+Build a Firmware
+~~~~~~~~~~~~~~~~
 
-The next schema presents the steps to build a MicroEJ Mono-Sandbox
-Firmware (previously known as MicroEJ Single-app Firmware).  The steps
+The next schema presents the build flow of a Mono-Sandbox
+Firmware (previously known as a MicroEJ Single-app Firmware). The steps
```
[the remark this answers is not in the document]
Comment (tdelhomenie-microej) [reply Tutorials/tutorialUnderstandMicroEJFirmwareBuild.rst:180]: Good point.
But that is quite confusing, the differences are very subtile.
@frivieremicroej cannot we simplify this by merging terms for example?

[the remark this answers is not in the document]
Comment (frivieremicroej) [reply Tutorials/tutorialUnderstandMicroEJFirmwareBuild.rst:180]: I will submit the request for the next glossary meeting. let it as is so far

File: Tutorials/tutorialUnderstandMicroEJFirmwareBuild.rst [hunk 5]
```diff
@@ -1,130 +1,124 @@
...
+~~~~~~~~~~~~
 
-A MicroEJ Architecture contains the runtime port to a target
-instruction set (ISA) and a C compiler (CC) and MicroEJ Foundation
-Libraries.
+A :ref:`MicroEJ Architecture <architecture_overview>` contains the runtime port to a target
+instruction set (ISA), a C compiler (CC) and Foundation Libraries.
 
-The MicroEJ Architectures are distributed into two formats:
+The Architectures are distributed in two versions:
```
Comment (frivieremicroej) [inline Tutorials/tutorialUnderstandMicroEJFirmwareBuild.rst:31]:
```suggestion
Architectures are distributed in two versions:
```

File: Tutorials/tutorialUnderstandMicroEJFirmwareBuild.rst [hunk 6]
```diff
@@ -1,130 +1,124 @@
...
-* EVAL: evaluation license with runtime limitations (explained in SDK
-  developer guide).
+* :ref:`Evaluation Architectures <evaluation_license>`: license with runtime limitations (explained in the :ref:`Application Developer Guide <limitations>`).
 
-* PROD: production license (only MicroEJ sales & Customer Care team
-  distribute this version).
+* :ref:`Production Architectures <production_license>`: license suitable for production.
 
-The supported MicroEJ Architectures are listed here
+A selection of supported embedded architectures can be found here:
```
Comment (frivieremicroej) [inline Tutorials/tutorialUnderstandMicroEJFirmwareBuild.rst:37]:
```suggestion
A selection of supported embedded Architectures can be found here:
```

**Review by frivieremicroej — CHANGES_REQUESTED** [review]

File: Tutorials/tutorialUnderstandMicroEJFirmwareBuild.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (tdelhomenie-microej) [reply Tutorials/tutorialUnderstandMicroEJFirmwareBuild.rst:180]: There is a "glossary meeting" ?

**Review by tdelhomenie-microej — APPROVED** [review]

**Review by frivieremicroej — APPROVED** [review]

**Pull request merged**